node-test/src/test/scala/org/bitcoins/node/networking/peer/PeerMessageHandlerTest.scala

```diff
@@ -39,35 +39,44 @@ class PeerMessageHandlerTest
   implicit protected lazy val chainConfig: ChainAppConfig =
     cachedConfig.chainConf
 
-  behavior of "PeerHandler"
+  override def beforeAll(): Unit = {
+    implicit val chainConf: ChainAppConfig = cachedConfig.chainConf
```

Comment:
This is probably going to need to change as per work on #2943 , we probably don't want to block this work on that though.